### PR TITLE
Sebs/incompatible nom

### PR DESCRIPTION
**Pull request by sebs2837**
Update nom since the used nom version is about to be deprecated.

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -17,7 +17,7 @@ license = "MIT"
 name = "rogcat"
 readme = "README.md"
 repository = "https://github.com/flxo/rogcat"
-version = "0.4.6"
+version = "0.4.9-pre"
```
Comment (flxo) [inline Cargo.toml:20]: This is a huge jump. Think this should be 0.4.7-pre and additionally not part of this pr. I'll fix it.

File: src/cli.rs [hunk 2]
```diff
@@ -172,6 +172,12 @@ pub fn cli() -> App<'static, 'static> {
      //         .long("show-time-diff")
      //         .conflicts_with("output")
      //         .help( "Show the time difference between the occurence of equal tags in terminal output"))
+        .arg(Arg::with_name("dev")
```
Comment (flxo) [inline src/cli.rs:175]: can we get this in a seperate PR?

**Review by flxo — CHANGES_REQUESTED** [review]
Thank to the contribution. The parsing part looks good. Please split off the `dev` feature and remove the version update.

File: src/cli.rs (hunk 2 above)
Comment (sebs2837) [reply src/cli.rs:175]: your request is my command... I'll change it immediately

File: Cargo.toml (hunk 1 above)
Comment (sebs2837) [reply Cargo.toml]: I guess I got a bit to excited with those version numbers, I changed it to 0.4.7-pre hope that's fine for your

**Pull request merged**